typed-racket-lib/typed-racket/utils/prefab-c.rkt

```diff
@@ -160,6 +161,19 @@
             [else
```

Comment:
oh hey, the two `contract-stronger?`s above this line can be replaced with one `contract-equivalent?`